card/mod/edit/set/all/bridge/tab_views.rb

```diff
@@ -0,0 +1,73 @@
+format :html do
+  RELATED_ITEMS = [["children",       :children],
+                   # ["mates",          "bed",          "*mates"],
+                   # FIXME: optimize and restore
+                   ["references out", :refers_to],
+                   ["references in",  :referred_to_by]].freeze
+
+  view :engage_tab, wrap: { div: { class: "m-3 mt-4" } } do
+    [render_follow_section, discussion_section].compact
+  end
+
+  view :history_tab do
+    class_up "d0-card-body",  "history-slot"
+    voo.hide :act_legend
+    acts_bridge_layout card.history_acts
+  end
+
+  view :related_tab do
+    bridge_pills bridge_pill_items(RELATED_ITEMS, "Related")
+  end
+
+  view :rules_tab do
```

Comment:
need `unknown_ok` here, right?  (Because I think we want to be able to see rules for new cards)